qubes/api/admin.py

```diff
@@ -145,6 +145,33 @@ def vm_list(self):
             self._vm_line_strs(strs, vm)
         return ''.join(strs)
 
+    @qubes.api.method('admin.vm.GetAllData', no_payload=True,
```

Comment:
Sorry, this is too much. Even if we agree for `property.GetAll`, getting all info of all the VMs in one call is beyond what we want in Admin API.